.babelrc

```diff
@@ -0,0 +1,32 @@
+{
+  "presets": [
```

Comment:
You probably want to add `@parcel/babel-preset-env` in here, so that the browserslist in package.json is applied

Reply:
Ah good point, I forgot that defining the babelrc overrode that.